MeNext.Spotify/SpotifyMetadata/MusicServiceImpls/SpotifyAlbum.cs

```diff
@@ -4,6 +4,9 @@
 
 namespace MeNext.Spotify
 {
+    /// <summary>
+    /// Class that represents an album and spotify metadata.
```

Comment:
Not "and". An Album is metadata. "Class that represents an album" is all you need.